src/content/1.7/modules/core_updates/1.7.7.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changes in PrestaShop 1.7.7
+menuTitle: Changes in 1.7.7
+---
+
+# Notable changes in PrestaShop 1.7.7
+
+## Order view page layout modifications
+
+In this version the order page was migrated with Symfony. We used this migration as an opportunity to improve the page layout so that it meets the merchant needs.
+
+### Modified hooks
+
+These hooks **have been moved** and have new names, even though they are very similar to their former equivalent the layout has changed and thus the modules need to adapt their rendering:
+
+| Former hook                | New hook                  | Modification           |
+| -------------------------- |:------------------------- |:------------------------------------------------------------------------------------------------------ |
```

Comment:
Be aware that two or three dashes are enough for the markdown interpreter

Reply:
I know but I find it more readable when they are correctly aligned